en/docs/develop-components/deploy-an-application-with-buildpacks.md

```diff
@@ -249,4 +249,13 @@ Follow the guidelines below based on your language:
     e.g.,
 
     `web: java -jar target/sample.war` <br>
-`
+
+### Build Time Environment Variables
+
+You can configure the environment variables that are necessary to build the component using the Build Configurations Editor in the component Build page. This option is not available for Ballerina and WSO2 MI.
+
+These environment variables and values will be passed to buildpack during the build time. Hence, this option can be used to configure buildpack specific environment variables or enviromemt variables required for component build.
+
+For example, you can override the Maven command of `Java` buildpack using `GOOGLE_MAVEN_BUILD_ARGS` as the environment key and `clean install` as the value. 
```

Comment:
For example, if you want to override the Maven command of the **Java** buildpack, you can use `GOOGLE_MAVEN_BUILD_ARGS` as the environment key and `clean install` as the value.